cmake/multi_image.cmake

```diff
@@ -38,6 +38,10 @@ function(image_board_selection board_in board_out)
     set(${board_out} nrf9160_pca20035 PARENT_SCOPE)
     message("Changed board to secure nrf9160_pca20035 (NOT NS)")
 
+  elseif(${board_in} STREQUAL nrf5340_dk_nrf5340_cpuappns)
```

Comment:
Lets make a function for this.

Reply:
Refactored (no function though)

Reply:
Note that some boards have 'ns' as a suffix and some have '_ns' unfortunately.

Reply:
Can't we just check for boards ending with both `xns` and` x_ns`?